test/ReactWrapper-spec.js

```diff
@@ -549,6 +549,15 @@ describeWithDOM('mount', () => {
       expect(() => wrapper.find(null)).to.throw(Error);
     });
 
+    it('Should query attributes with spaces in their values', () => {
+      const wrapper = mount(
+        <div>
+          <h1 data-foo="foo bar">Hello</h1>
+        </div>
+      );
+      expect(wrapper.find('[data-foo="foo bar"]')).to.have.length(1);
```

Comment:
Can you add another test to verify that attribute queries still work without the attribute value? 

```js
wrapper.find("[data-foo]")
```